lib/logger/lib/logger/backends/console.ex

```diff
@@ -337,6 +337,10 @@ defmodule Logger.Backends.Console do
     retry_log({:put_chars, :unicode, output}, state)
   end
 
+  defp handle_io_reply({:error, {:no_translation, _encoding_from, _encoding_to} = error}, state) do
```

Comment:
FYI: This clause is already needed even before the OTP PR: See https://github.com/erlang/otp/blob/277c1d5bd088d8114395d1cf1b666539d42b86c7/lib/kernel/src/file_io_server.erl#L446